SingularityUI/app/selectors/requests/filterSelector.es6

```diff
@@ -74,9 +74,16 @@ export default createSelector([getRequests, getFilter, getUtilizations], (reques
       });
       filteredRequests = _.union(res1, res2).reverse();
     } else {
-      _.each(filteredRequests, (requestParent) => {requestParent.id = id.extract(requestParent);});
+      _.each(filteredRequests, (requestParent) => (requestParent.id = id.extract(requestParent)));
```

Comment:
Returning a value in `_.each` shouldn't do anything so I'm not sure why this change was necessary. The existing code is pretty unclear about what this loop actually does. There appears to be some sort of side effect with the `extract` function. Would it be easy to clean up the existing code?

Reply:
Hmm I did seem to make it less clear since now it's returning something and we don't use it. it was unclear to begin so I'll see if I can clean it up.

Reply:
`selectors/tasks` has similar code and it looks like it makes more sense in that file and less sense here.